examples/go_examples.md

```diff
@@ -28,7 +28,7 @@ script:
   - go test -coverprofile c.out ./...
 
 after_script:
-  - ./cc-test-reporter after-build -t go test -coverprofile=c.out --exit-code $TRAVIS_TEST_RESULT
+  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
No need to use the `-t` or `--input-type` flag over here because examples are using the default output `c.out `.